ProjectManagmentSystem/src/App.css

```diff
@@ -0,0 +1,77 @@
+.auth-container {
```

Comment:
The styles in this file will be global, if we want to scope it to certain components, we should be using module css instead.